### ISSUE TITLE
some headers show up really large

### ISSUE THREAD

**Issue by ghost** [paullinuxthemer/Nextwaita#6]
![image](https://user-images.githubusercontent.com/50459218/57477538-cce61a80-7290-11e9-8a8d-bb82352ede6b.png)

most are fine but some apps like terminal, firefox, and others have a really tall header. thank you so much for this theme btw, it's really great.

**Comment by Polisson** [comment]
Yes, I can confirm. It is relevant to HiDPI screen scaling (like 200% on 4K displays). On default 100% scaling problem seems to be imperceptible.
